src/Symfony/Component/Messenger/composer.json

```diff
@@ -28,6 +28,7 @@
         "symfony/http-kernel": "^6.4|^7.0",
         "symfony/process": "^6.4|^7.0",
         "symfony/property-access": "^6.4|^7.0",
+        "symfony/lock": "^6.4|^7.0",
```

Comment:
Please add a `"symfony/lock": "<6.4"` entry to the `conflict` section.

Reply:
Done